org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/extensions/catalog/CatalogUtils.java

```diff
@@ -30,13 +30,21 @@
 	/**
 	 * The catalog entries that have a 'uri' attribute
 	 */
-	private static final Collection<String> CATALOG_NAMES = Arrays.asList("public", "system", "uri", "systemSuffix",
+	private static final Collection<String> HAS_URI_ATTRIBUTE = Arrays.asList("public", "system", "uri", "systemSuffix",
 			"uriSuffix");
 
+	/**
+	 * The catalog entries that have a 'catalog' attribute
+	 */
+	private static final Collection<String> HAS_CATALOG_ATTRIBUTE = Arrays.asList("delegatePublic", "delegateSystem",
+			"delegateUri", "nextCatalog");
+
 	private static final String CATALOG_ENTITY_NAME = "catalog";
 
 	private static final String URI_ATTRIBUTE_NAME = "uri";
 
+	private static final String CATALOG_ATTRIBUTE_NAME = CATALOG_ENTITY_NAME;
```

Comment:
I think it should be better to assign the constant to "catalog" even it's the same name, it's not the same thing.